docs/fedora-releases-guide/what-to-know.md

```diff
@@ -0,0 +1,106 @@
+---
+title: What maintainers should know before adopting Packit for downstream release automation 
+sidebar_position: 5
+---
+# What maintainers should know before adopting Packit for downstream release automation 
+
+## Responsibilities
+
+Packit is designed to **assist** maintainers, not to **replace** them.
+
+Maintainers retain key responsibilities:
+- reviewing and merging Packit-created pull requests
+- maintaining package quality
+- ensuring updates don't break dependant packages
+
+:::tip `version_update_mask`
+
+The default Packit onboarding configuration enables automated releases only for `rawhide`. 
+
+When extending the automation to branched Fedoras and EPELs, consider using [`version_update_mask`](https://packit.dev/docs/configuration#version_update_mask) to prevent introducing breaking changes into stable distributions.
+
+:::
+
```

Comment:
Ideally, I would put here another warning about the **spamming**.
But I am not really sure what the users can do about it?
@lachmanfrantisek do you know what users can do to avoid creating too much spamming?